.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
     - node_js: '8'
       script:
         - yarn build
-        - yarn test
+        - 'yarn test && yarn coverage'
```

Comment:
Could you move this to the `yarn test` in the build above? The "normal" build config is node 10, and this build config is the one to make sure I didn't break node 8.